src/chainstate/stacks/boot/pox.clar

```diff
@@ -327,23 +327,35 @@
     (and (>= lock-period MIN-POX-REWARD-CYCLES) 
          (<= lock-period MAX-POX-REWARD-CYCLES)))
 
-;; Register a PoX address for one or more reward cycles, and set how many uSTX it locks up initially.
-;; Will fail if the PoX address is already registered in one of the reward cycles.
-;; Will fail if the number of uSTX is beneath the lowest allowed Stacking threshold at the time of the call.
-(define-private (register-pox-addr-checked (pox-addr (tuple (version (buff 1)) (hashbytes (buff 20))))
-                                           (amount-ustx uint)
-                                           (first-reward-cycle uint)
-                                           (num-cycles uint))
-    (let (
-        (ustx-min (get-stacking-minimum))
+;; Evaluate if a participant can stack an amount of STX for a given period.
+;; This method is designed as a read-only method so that it can be used as 
+;; a set of guard conditions, but also via a read-only RPC call that can be
```

Comment:
"and also as a read-only RPC call".  Also, "beforehand" is one word.

Reply:
Apologies, addressed with https://github.com/blockstack/stacks-blockchain/pull/1899/commits/c1fd294952c27c9fc09f8e40171c28f0c2de254d